code/modules/shuttle/marine_dropship.dm

```diff
@@ -167,6 +169,12 @@
 		return
 	// pull the shuttle from datum/source, and state info from the shuttle itself
 	SEND_GLOBAL_SIGNAL(COMSIG_GLOB_DROPSHIP_TRANSIT)
+	if(first_landing)
+		for(var/mob/living/carbon/human/human AS in GLOB.alive_human_list)
```

Comment:
Optional:
Check for gamemode here? will stop any funnies if an admin moves the shuttle for an event or whatever